explorer/src/lib/stores/appStore.js

```diff
@@ -3,39 +3,68 @@ import { browser } from "$app/environment";
 
 /** @type {NetworkOption[]}*/
 const networks = [
-  { label: "Testnet", value: import.meta.env.VITE_DUSK_TESTNET_NODE },
-  { label: "Devnet", value: import.meta.env.VITE_DUSK_DEVNET_NODE },
+  { label: "Local", value: new URL("/", import.meta.url) },
+  {
+    label: "Devnet",
+    value: new URL(
+      `${window.location.protocol}${import.meta.env.VITE_DUSK_DEVNET_NODE}`
+    ),
+  },
+  {
+    label: "Testnet",
+    value: new URL(
+      `${window.location.protocol}${import.meta.env.VITE_DUSK_TESTNET_NODE}`
+    ),
+  },
+  {
+    label: "Mainnet",
+    value: new URL(
+      `${window.location.protocol}${import.meta.env.VITE_DUSK_TESTNET_NODE}`
```

Comment:
I still think we can have this as `import.meta.env.VITE_DUSK_MAINNET_NODE`, even if the url is still unknown. It would then require change in the ENV only, rather than a new PR to change it here, in the code as well.